### ISSUE TITLE
Move to a Chrome app instead of extension

### ISSUE THREAD

**Issue by diracdeltas** [yahoo/end-to-end#1]
The XSS potential with running as an extension makes a lot of people rightfully nervous. There is an upstream Google branch that moves to an app instead. We should do that too.

**Comment by diracdeltas** [comment]
Started working on this and realized this is blocked on https://code.google.com/p/chromium/issues/detail?id=460797 :(

**Comment by koto** [comment]
Yeah, I encountered the same showstoppeer in Jan :/ I commented, maybe we get get this going...
